custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -2279,6 +2279,13 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
         vm.getDataForStep(vm.step);
     });
 
+    vm.isOlderThan90Days = function () {
```

Comment:
can you rename the function, its no longer doing what it was doing. It looks currently its checking if backend has returned any images or not